app.py

```diff
@@ -111,11 +129,22 @@ def downloader_main(output_dir, do_overwrite, audio_book_url):
 
     for url, fname in playlist:
         click.echo('Downloading chapter "{}"'.format(fname))
-        with open('{}.mp3'.format(os.path.join(full_path_dir, fname)), 'wb') as outfile:
+        if one_file:
+            file_name_and_path = '{}.mp3'.format(
+                get_audiobook_name(audio_book_url)
```

Comment:
We lost the `full_path_dir` here (mind combination of path and `--onefile` option).

Reply:
Okay, I was thinking to write it without additional directory, but this is also an option. I will fix it.